packages/core/addon/models/bard-request/request.js

```diff
@@ -551,7 +551,9 @@ export default Fragment.extend(Validations, {
   clone() {
     let clonedRequest = this.toJSON(),
       store = this.store,
-      metadataService = this.metadataService;
+      metadataService = this.metadataService,
+      dimensionTypeMap = Object.fromEntries(this.dimensions.map(dim => [dim.dimension.id, dim.dimension.valueType])),
```

Comment:
`Object.fromEntries` 

nice

Reply:
I know right? so much better than `reduce` I been waiting for the time I needed to make an object from an array for this 😄